ITISFoundation/osparc-simcore
# 🔨 Added new 'up' target for front-end development

## What do these changes do?
See title. Some nice Docker wizardry from @pcrespov 






## How to test
`make build`
`make up-devel-frontend`
That should start all services except `static-webserver` in prod mode. `static-webserver` remains in dev mode to have the QX files mounted in.
